src/tools/CMakeLists.txt

```diff
@@ -16,6 +16,14 @@ else()
 endif()
 install(TARGETS rados DESTINATION bin)
 
+set(neorados_srcs
+  neorados.cc)
+add_executable(neorados ${neorados_srcs})
+
+target_link_libraries(neorados libRADOS global Boost::coroutine ${CMAKE_DL_LIBS})
+#install(TARGETS neorados DESTINATION bin)
```

Comment:
as a test program, it might make sense to add to the ceph-test package (the others prefixed with ceph-test will automatically be picked up), and add something that runs them to the rados suite